src/Pages/RequestPage/Component/RequsetList.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component, Fragment } from "react";
+import styled from "styled-components";
+
+class RequsetList extends Component {
+  state = {
+    requestList: [],
+  };
+
+  componentDidMount() {
+    fetch("Data/userData.json")
+      .then((res) => res.json())
+      .then((result) => this.setState({ requestList: result }));
+  }
+
+  render() {
+    const { requestList } = this.state;
+
+    return (
+      <RequestListed>
+        {requestList &&
+          requestList.map((item, index) => {
+            return (
+              <Fragment key={index}>
+                <RequsetContainer>
+                  <UserContainer>
+                    <img src={item.userimg} alt="유저이미지" />
+                    <Description>
+                      <P>{item.userName}</P>
+                      <Span>{item.category}</Span>
+                      <Span>{item.address}</Span>
+                      <Spandes>
+                        {item.description.length > 58
+                          ? item.description.slice(0, 58) + "..."
+                          : item.description}
+                      </Spandes>
+                    </Description>
+                    <Information>
+                      <Span>{item.time}</Span>
+                      <button>삭제</button>
```

Comment:
```suggestion
                      <button type="button">삭제</button>
```
button 태그에 타입을 넣지 않아도 동작은 하지만, 넣지 않았을 때 문제가 생길 여지가 있습니다. 만약 button 태그에 타입을 넣지 않았을 때 어떤 일이 발생하는지 검색해보시고 댓글로 조사한 내용을 남겨주세요!

Reply:
버튼 태그에 속성이 없을 땐 submit 기능을 수행할 수 있기 때문에 기능 동작을 원할 때는 type="button" 을 사용해야 하는거 같습니다.